src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/ParameterNames.java

```diff
@@ -0,0 +1,45 @@
+package com.icthh.xm.tmf.ms.communication.messaging.handler;
+
+public final class ParameterNames {
```

Comment:
Suggest renaming class to `SmppParameterNames`

Reply:
There are not only smpp parameters but MobileApp as well. 
We use _VALIDITY_PERIOD_ in both requests.
The intent is to have unified parameters for all types of communication messages.